standardization/src/test/scala/za/co/absa/enceladus/standardization/interpreter/stages/TypeParserSuite.scala

```diff
@@ -25,9 +25,9 @@ class TypeParserSuite extends FunSuite with SparkTestBase {
   private implicit val udfLib: UDFLibrary = new za.co.absa.enceladus.utils.error.UDFLibrary
 
   test("Test standardize with sourcecolumn metadata") {
-    val structFieldNoMetadata = StructField("a", IntegerType)
-    val structFieldWithMetadataNotSourceColumn = StructField("b", IntegerType, nullable = false, new MetadataBuilder().putString("meta", "data").build)
-    val structFieldWithMetadataSourceColumn = StructField("c", IntegerType, nullable = false, new MetadataBuilder().putString("sourcecolumn", "override_c").build)
+    val structFieldNoMetadata = StructField("a", StringType)
+    val structFieldWithMetadataNotSourceColumn = StructField("b", StringType, nullable = false, new MetadataBuilder().putString("meta", "data").build)
+    val structFieldWithMetadataSourceColumn = StructField("c", StringType, nullable = false, new MetadataBuilder().putString("sourcecolumn", "override_c").build)
```

Comment:
Why do this?
+ in general, this suite seems a bit weird in the way of what it checks.